content/en/press-kit.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Press kit
+sidebar: false
+---
+
+We would like to make it easy for you to include the NumPy brand identity in your next academic paper, course materials or presentation. 
+
+You will find a high resolution logo of NumPy [here](https://numpy.org).
+Note that by using the numpy.org resources, you accept [NumPy Code of Conduct](https://numpy.org/code-of-conduct).
```

Comment:
minor: suggest changing to "... accept the [NumPy ..."

Reply:
Done. Thank you for catching it, @rgommers!